control-plane/roles/nsq/templates/nsq.yaml

```diff
@@ -73,7 +73,7 @@ spec:
         args:
           - /nsqadmin
           - -log-level={{ nsq_log_level }}
-          - -lookupd-http-address=nsq-lookupd:4161
+          - -lookupd-http-address={{ api_nsq_lookupd }}
```

Comment:
This uses a variable of another role, which violates role separation. 

What was the reason for you to change it? The nsq lookupd service has a static name as well. Can this even work with another http address without special DNS records?